dataactcore/aws/s3UrlHandler.py

```diff
@@ -132,3 +132,14 @@ def get_file_urls(self, bucket_name, path):
                 urls[file_name] = url
 
         return urls
+
+    @staticmethod
```

Comment:
I think it's time we rename this file since it's doing more than simple URL manipulation.